service.go

```diff
@@ -29,7 +29,7 @@ type Service struct {
 func NewService(env Environment) (*Service, error) {
 	fi, err := os.Stat(env.Directory)
 	if err != nil {
-		return nil, fmt.Errorf("directory %s unavailable: %s", env.Directory, err.Error())
+		return nil, fmt.Errorf("directory '%s' unavailable: %s", env.Directory, err.Error())
```

Comment:
Why not, but I would like this in a separate MR. Nitpicking, but I prefer to have single-purpose MRs.

Reply:
I'll keep that in mind.